config.go

```diff
@@ -65,11 +66,12 @@ func CreateConfig(path string) error {
 			config.LoginHost = "my.rightscale.com"
 		}
 		hostDef = fmt.Sprintf(" (%v)", config.LoginHost)
+		refreshTokenDef = fmt.Sprintf(" (%v)", config.RefreshToken)
```

Comment:
Just noticed this. Refresh token is as sensitive as the password. Can we not print it on the console?